frame/composable-traits/src/lending/math.rs

```diff
@@ -356,19 +358,15 @@ impl DoubleExponentModel {
 
 impl InterestRate for DoubleExponentModel {
 	fn get_borrow_rate(&mut self, utilization: Percent) -> Option<Rate> {
-		let polynomial_0: FixedU128 = FixedU128::one();
-		let result_0: Rate = Rate::zero();
-		let res =
-			self.coefficients
-				.iter()
-				.try_fold((result_0, polynomial_0), |(res, poly), coeff| {
-					let coefficient = FixedU128::from_inner(u128::from(*coeff));
-					let result = res.checked_add(&coefficient.checked_mul(&poly)?)?;
-					let polynomial = poly.checked_mul(&utilization.into())?;
-					Some((result, polynomial))
-				});
-		res.map(|(r, _p)| r.checked_div(&FixedU128::from_inner(EXPECTED_COEFFICIENTS_SUM.into())))
-			.unwrap()
+		// as per model 
```

Comment:
Is there any difference between previous implementation vs this new code? or is it just to make it easy to read? @dzmitry-lahoda

Reply:
Thanks @dzmitry-lahoda for explaining the diff offline on Slack. Marking this resolved.